daemon/main.c

```diff
@@ -1497,8 +1305,23 @@ int main(int argc, char **argv) {
     info("netdata initialization completed. Enjoy real-time performance monitoring!");
     netdata_ready = 1;
 
+    set_late_global_environment();
+
     send_statistics("START", "-",  "-");
 
+    //check if ANALYTICS needs to start
+    if (netdata_anonymous_statistics_enabled==1) {
```

Comment:
please add space before and after `==`.

Reply:
Fixed, thanks Thiago!

Reply:
Hello @MrZammler ,

I cloned your PR now and I cannot see the space, please, take a look on this one more time.

Thanks!

Reply:
Done, thanks and sorry for the mixup!